libs/composer/composer.json

```diff
@@ -35,7 +35,6 @@
 		"sabre/dav": "~3.2.2",
 		"symfony/console" : "^4.2",
 		"slim/slim": "^3.11",
-		"symfony/console" : "^4.2",
```

Comment:
Das sollte dann, glaub ich, nicht weg.

Reply:
Es steht zwei Zeilen drüber bereits redundant, von daher ;-). Es gehört aber nicht unbedingt in den PR hier :-).

Reply:
Sorry! Nicht gesehen.